package.json

```diff
@@ -17,6 +17,7 @@
     "@actions/core": "^1.2.3",
     "@actions/tool-cache": "^1.3.1",
     "download": "^7.1.0",
-    "semver": "^7.1.3"
+    "semver": "^7.1.3",
+    "typed-rest-client": "^1.7.2"
```

Comment:
Is this not coming as a part of `@actions/core` anymore?

Reply:
Not sure about this. But, is there any harm in having this dependency explicitly? Isn't it safer?

Reply:
It is, was just curious